lib/livebook/notebook.ex

```diff
@@ -101,6 +101,57 @@ defmodule Livebook.Notebook do
     put_in(notebook.setup_section.cells, [%{cell | id: Cell.setup_cell_id()}])
   end
 
+  @doc """
+  TODO
+  """
+  @spec move_output_to_canvas(t(), Cell.t(), Section.t()) :: t()
+  def move_output_to_canvas(notebook, cell, section) do
+    notebook
+    |> update_in(
+      [
+        Access.key(:sections),
+        access_by_id(section.id),
+        Access.key(:cells),
+        access_by_id(cell.id)
+      ],
+      fn cell ->
+        %{cell | output_location: %{x: 0, y: 0, w: 4, h: 2}}
```

Comment:
Is there a way to figure out the size of an output?

Reply:
Not really, most outputs don't have a specific size and just take whatever width they have, so we need to allocate a specific space.